cmd/action/docker.go

```diff
@@ -186,6 +187,7 @@ func (c *Container) Wait(ctx context.Context, timeout time.Duration) error {
 			if err := db.PingContext(ctx); err != nil {
 				continue
 			}
+			fmt.Println("Service is ready to connect!")
```

Comment:
Can we remove these printing from the CLI, or at least enable them in verbose mode? It doesn't feel UNIX-ish and will also add noise when redirecting stdout (e.g. atlas ... > out.sql).

Reply:
Currently the exec commands to spit out to os.Stdout as well, we would enable that as well. 

Another thing: we can print to stderr instead, in that case redirect would work as expected.